review/src/main/resources/application.yml

```
@@ -38,3 +44,8 @@ review:
       cron: 0 */5 * * * *
       batch-size: 100
       recent-minutes: 5
+
+distributed-lock:
+  wait-time: 0
+  lease-time: 3
+  time-unit: MINUTES
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)